src/pages/lesson-details/LessonDetails.tsx

```diff
@@ -78,6 +89,50 @@ const LessonDetails = () => {
     })
   }
 
+  const getCooperation = useCallback(() => {
+    return cooperationService.getCooperationById(id)
+  }, [id])
+
+  const { data: cooperation } = useQuery({
+    queryFn: getCooperation,
+    queryKey: ['cooperation', id]
+  })
+
+  useEffect(() => {
+    if (!cooperation?.sections) return
```

Comment:
Add braces here, please

Reply:
fixed

Reply:
Not fixed, there are no braces currently. It should be like this: 
```suggestion
    if (!cooperation?.sections) {
      return
    }
```